tripgo.swagger.yaml

```diff
@@ -89,27 +89,37 @@ paths:
           schema:
             $ref: '#/definitions/RegionInfoResponse'
 
-  /regionOperators.json:
+  /operatorInfo.json:
     post:
       tags:
         - Configuration
-      summary: List of TSPs per region.
+      summary: List of TSPs for a particular region.
       description:
-        Retrieves information about covered transport service providers for a specified region plus some global information.
+        Retrieves detailed information about covered transport service providers for a specified region.
       parameters:
         - name: input
           in: body
           schema:
             type: object
+            description: Use `modes` to filter by tranport modes. Only results for the given modes will be retuned. When ´full´ paramater is false, it will retrieve a list of operator names for the given region and ´modes´ and ´onlyRealTime´ if any. You can also retrieve information for a particular operator using ´operator´, providing its name as it was given before as a list member in this very same endpoint. ´onlyRealTime´ parameter to be used only when ´full´ is set false. The use of filters like ´modes´, ´onlyRealTime´ and/or ´operator´ could lead to a empty response. In order to prevent this, make sure such combinatios are posible, using previously fetched information.
             properties:
               region:
                 type: string
-                description: Region name/code from `regions.json`
+                description: Region name/code from `regions.json`.
               modes:
                 type: array
                 items:
                   type: string
-                description: Public transit modes (pt_pub_*) for which results should be returned. [Mode identifier](http://skedgo.github.io/tripgo-api/site/advanced/#mode-identifiers) strings like `pt_pub` or `pt_pub_subway`. In absence, all modes are included. An empty array means no transitModes array is returned.
+                description: Public transit modes (pt_pub_*) for which results should be returned. [Mode identifier](http://skedgo.github.io/tripgo-api/site/advanced/#mode-identifiers) strings like `pt_pub` or `pt_pub_subway`. In absence, all modes are included.
+              operator:
+                type: string
+                description: Operator name to retrieve information from. When ´full´ parameter is false, this field is ignored. THIS COULD BE AN ARRAY TOO.
+              onlyRealTime:
+                type: boolean
+                description: Boolean to filter only operators with Real Time support. Only valid when ´full´ parameter is false. When true, returned operator list will contain only TSP with Real Time service.
+              full:
+                type: boolean
+                description: Boolean if result should include full information for operator/s or just a list with all their names ignoring ´mode´ array in response. False value by default.
```

Comment:
sorry i got confused by "list or not".  so it is always a list of objects containing at least name, yes?